server/routes/auth/local.js

```diff
@@ -21,7 +20,7 @@ router.get('/twoFactor', async (req, res) => {
     res.redirect('/');
   } else if (req.user) {
     const transporter = createTransport();
-    await generateToTPSecret(req.user.dataValues.email, transporter);
+    await req.user.generateToTPSecret(req.user.dataValues.email, transporter);
```

Comment:
If the email is just coming from req.user.dataValues.email, do we need to pass it in as a parameter vs. accessing it on `this`?

Reply:
Do we also need to pass in the transporter? Or can that just be required in User.js?